src/current/v24.3/ldap-authentication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Configure LDAP Authentication
+summary: Learn how to configure CockroachDB to use LDAP for user authentication with directory services like Active Directory.
+toc: true
+---
+
+CockroachDB supports authentication and authorization using LDAP (Lightweight Directory Access Protocol) with services such as Active Directory. This allows you to integrate your existing directory service with CockroachDB for centralized user management and access control.
+
+This page describes how to configure CockroachDB user authentication using LDAP. You can also configure user authorization (role-based access control) with LDAP, to set users' CockroachDB roles based on group memberships in the directory, but this is not required.
+
+## Overview
+
+LDAP authentication in CockroachDB uses a 'search and bind' approach:
```

Comment:
```suggestion
LDAP authentication in CockroachDB uses a "search and bind" approach:
```
I've never heard this specific terminology before. Worth linking to a wiki or similar?